bundles/org.openhab.binding.amazonechocontrol/src/main/resources/ESH-INF/thing/thing-types.xml

```diff
@@ -1,530 +1,530 @@
-	</channel-type>
-	<channel-type id="lastVoiceCommand" advanced="true">
-		<item-type>String</item-type>
-		<label>Last Voice Command</label>
-		<description>Last voice command spoken to the device. Writing to the channel starts voice output.</description>
-		<state readOnly="false"/>
-	</channel-type>
-	<channel-type id="mediaProgress" advanced="false">
-		<item-type>Dimmer</item-type>
-		<label>Media Progress</label>
-		<description>Media progress in percent</description>
-		<state readOnly="false"/>
-	</channel-type>
-	<channel-type id="mediaProgressTime" advanced="true">
-		<item-type>Number:Time</item-type>
-		<label>Media Play Time</label>
-		<description>Media play time</description>
-		<state readOnly="false" pattern="%d %unit%"/>
-	</channel-type>
-	<channel-type id="mediaLength" advanced="true">
-		<item-type>Number:Time</item-type>
-		<label>Media Length</label>
-		<description>Media length</description>
-		<state readOnly="true" pattern="%d %unit%"/>
-	</channel-type>
-	<channel-type id="startCommand">
-		<item-type>String</item-type>
-		<label>Start</label>
-		<description>Start information (Write only)</description>
-		<state pattern="String" readOnly="false">
-			<options>
-				<option value="Weather">Weather</option>
-				<option value="Traffic">Traffic</option>
-				<option value="GoodMorning">Good morning</option>
-				<option value="SingASong">Song</option>
-				<option value="TellStory">Story</option>
-				<option value="FlashBriefing">Flash briefing</option>
-			</options>
-		</state>
-	</channel-type>
-	<channel-type id="notificationVolume" advanced="true">
-		<item-type>Dimmer</item-type>
-		<label>Notification Volume</label>
-		<description>Notification Volume</description>
-		<state readOnly="false"/>
-	</channel-type>
-	<channel-type id="ascendingAlarm" advanced="true">
-		<item-type>Switch</item-type>
-		<label>Ascending Alarm</label>
-		<description>Ascending alarm up to the configured volume</description>
-		<state readOnly="false"/>
-	</channel-type>
-	<channel-type id="nextReminder" advanced="true">
-		<item-type>DateTime</item-type>
-		<label>Next Reminder</label>
-		<description>Next Reminder</description>
-		<state readOnly="true"/>
-	</channel-type>
-	<channel-type id="nextAlarm" advanced="true">
-		<item-type>DateTime</item-type>
-		<label>Next Alarm</label>
-		<description>Next alarm</description>
-		<state readOnly="true"/>
-	</channel-type>
-	<channel-type id="nextMusicAlarm" advanced="true">
-		<item-type>DateTime</item-type>
-		<label>Next Music Alarm</label>
-		<description>Next music alarm</description>
-		<state readOnly="true"/>
-	</channel-type>
-	<channel-type id="nextTimer" advanced="true">
-		<item-type>DateTime</item-type>
-		<label>Next Timer</label>
-		<description>Next timer</description>
-		<state readOnly="true"/>
-	</channel-type>
-</thing:thing-descriptions>
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Please check what happened to this file.

Reply:
look above